semcore/utils/src/assignProps.ts

```diff
@@ -15,7 +17,7 @@ export function callAllEventHandlers(...fns) {
 
 export function assignHandlers(props, source) {
   return Object.keys(source).reduce((proxySource, propName) => {
-    if (typeof source[propName] === 'function' && propName.startsWith('on')) {
+    if (typeof source[propName] === 'function' && handlerRex.test(propName)) {
```

Comment:
Use `.startsWith`. Regular expression seems to be unacceptably heavy tool here like blowing away a dandelion with a gun 🔫

Reply:
I'm afraid of false positives on properties like "onpreview={React.FC}"

Reply:
I weighed all the pros and cons and leave the regex because this code is very ugly
```
function validateHandlerName(handlerName: string) {
  if (handlerName[2]) {
    return handlerName.startsWith('on') && handlerName[2] === handlerName[2]?.toUpperCase();
  }
  return false
}
```

Reply:
<img width="1003" alt="Screenshot 2022-08-17 at 12 08 15" src="https://user-images.githubusercontent.com/4582799/185080783-752c3967-8b5d-4f66-98fd-f49e3e7f340c.png">
The difference in performance is not very big.

Reply:
The rule of thumb is just to do not use regex in performance bottlenecks where you are able not to use it